ipatests/test_integration/test_external_ca.py

```diff
@@ -21,12 +21,13 @@
 from ipatests.pytest_plugins.integration import tasks
 from ipatests.test_integration.base import IntegrationTest
 from ipaplatform.paths import paths
-
+import os
```

Comment:
Standard library imports belong in the top import section. Please put it before ``import re``.